Control_EnvironmentGUI.py

```diff
@@ -0,0 +1,132 @@
+from tkinter import *
+import pygame
+
+
+GREEN = '#0c871b'
+
+
+class EnvironmentControl:
+    def __init__(self, *args, **kwargs):
+        """
+        This is the initialization of the environment window controls that
+        will need to be kept track of throughout the run
+        """
+        self.speed = 1
+        # TODO: Make these correspond to the correct values in setup_environmentGUI
+        self.height = 1
+        self.width = 1
+
+    def set_speed(self, speed):
+        """
+        This is where the speed of the environment is set from the settings window
+
+        :param speed: The speed of the environment. Ex. "2x" or ".5x"
+        :param type: string
+        """
+        self.speed = int(speed[:-1])
+
+    def set_size(self, width, height):
+        """
+        This is where the size is changed together. This may be called at a
+        different time from the individual setter functions
+
+        :param width: Width of the window
+        :param type: int
+
+        :param height: Height of the window
+        :param type: int
+        """
+        self.height = height
+        self.width = width
+
+    def start(self):
+        """
+        This is where the environment is started up again when it is selected in the environment window
+        """
+        print("start")
+        pass
+
+    def stop(self):
+        """
+        This is the call to stop whenever it is clicked in the environment window
+        """
+        print("stop")
+        pass
+
+    def set_width(self, width):
+        """
+        This is where the width is set and adjusted from the settings window
+
+        :param width: The width of the window
+        :Param type: int
+        """
+        print(f"width: {width}")
+        self.width = width
+
+    def set_height(self, height):
+        """
+        This is where the height is set and adjusted from the settings window
+
+        :param height: The height of the window
+        :param type: int
+        """
+        print(f"height: {height}")
+        self.height = height
+
+    def set_temperature(self, temperature):
```

Comment:
on the c++ side we are moving to using an array of the environmental variables. We'll probably have like 4, but it would probably be smart to have a general setter for any of the environment variables that takes an argument for which variable it sets. You can set up an enum that matches ours eventually, that hasn't been built on the c++ side yet so we don't exactly know which variable names values 0, 1, 2, and 3 will map to yet